### DISCUSSION TITLE
Delete user data extension + Firebase stripe extension causes an error

### DISCUSSION THREAD

**Discussion by tomasconnor** [firebase/extensions#1254] [Q&A]
I have Delete user data extension and Firebase stripe extension. If I delete a user in the auth section, the Delete user data extension deletes all records about the user. At the same time, I have auto delete enabled in the Firebase stripe extension settings. Everything is deleted correctly, including the customer object in the Stripe dashboard, however, a 404 error appears in the Stripe error log.

{
  "error": {
    "code": "resource_missing",
    "doc_url": "https://stripe.com/docs/error-codes/resource-missing",
    "message": "No such customer: 'cus_Ma34mdpBTy0zFv'",
    "param": "id",
    "type": "invalid_request_error"
  }
}

Does anyone else have the same problem?

**Comment by cabljac** [comment]
This seems like expected behaviour to me, if delete-user-data AND the stripe extension are both trying to delete some data, one of them (stripe in this case) isn't going to find the data to delete. We could downgrade it to a warning perhaps
